src/components/Login/LocalLogin.tsx

```diff
@@ -0,0 +1,152 @@
+  loginerror: 'Something went wrong when trying to sign in',
+  loggingin: 'Logging in...',
+  login: 'Login',
+  goback: 'Go back',
+});
+
+interface LocalLoginProps {
+  goBack: () => void;
+  revalidate: () => void;
+}
+
+const LocalLogin: React.FC<LocalLoginProps> = ({ goBack, revalidate }) => {
+  const intl = useIntl();
+  const [loginError, setLoginError] = useState<string | null>(null);
+
+  const LoginSchema = Yup.object().shape({
+    email: Yup.string()
+      .email()
+      .required(intl.formatMessage(messages.validationemailrequired)),
+    password: Yup.string().required(
+      intl.formatMessage(messages.validationpasswordrequired)
+    ),
+  });
+
+  return (
+    <Formik
+      initialValues={{
+        email: '',
+        password: '',
+      }}
+      validationSchema={LoginSchema}
+      onSubmit={async (values) => {
+        try {
+          await axios.post('/api/v1/auth/local', {
+            email: values.email,
+            password: values.password,
+          });
+        } catch (e) {
+          setLoginError(intl.formatMessage(messages.loginerror));
+        } finally {
+          revalidate();
+        }
+      }}
+    >
+      {({
+        errors,
+        touched,
+        isSubmitting,
+        // values,
+        isValid,
+        // setFieldValue,
+      }) => {
+        return (
+          <>
+            <Form>
+              <div className="mt-6 sm:mt-5 sm:border-t sm:border-gray-800">
+                <label
+                  htmlFor="email"
+                  className="block my-1 text-sm font-medium leading-5 text-gray-400 sm:mt-px"
+                >
+                  {intl.formatMessage(messages.email)}
+                </label>
+                <div className="mt-1 mb-2 sm:mt-0 sm:col-span-2">
+                  <div className="flex max-w-lg rounded-md shadow-sm">
+                    <Field
+                      id="email"
+                      name="email"
+                      type="text"
+                      placeholder="name@example.com"
+                      className="text-white flex-1 block w-full min-w-0 transition duration-150 ease-in-out bg-gray-700 border border-gray-500 rounded-md form-input sm:text-sm sm:leading-5"
+                    />
+                  </div>
+                  {errors.email && touched.email && (
+                    <div className="mt-2 text-red-500">{errors.email}</div>
+                  )}
+                </div>
+                <label
+                  htmlFor="password"
+                  className="block my-1 text-sm font-medium leading-5 text-gray-400 sm:mt-px"
+                >
+                  {intl.formatMessage(messages.password)}
+                </label>
+                <div className="mt-1 mb-2 sm:mt-0 sm:col-span-2">
+                  <div className="flex max-w-lg rounded-md shadow-sm">
+                    <Field
+                      id="password"
+                      name="password"
+                      type="password"
```

Comment:
should this be a password field? I feel like we want to see what we are typing here because it doesn't have a confirmation and it's for another user. What do you think?

Reply:
I assume you meant to referr to the password field in the create modal and not on the login screen. Either way I think it should be a password field.

You might have other people who can see what you're doing on the screen that are not supposed to know the password. I also think it needs to be a password field in order to be correctly configured for password managers and the like.

Keeping it like it is for now and maybe if anyone has any objections this discussion can be had in an issue.

Reply:
Sorry yes, I meant the creation modal. My mistake. In the future let's add a checkbox that toggles showing the password.

Just for note, you don't need a password field for password managers, you instead can use the html attribute `autocomplete`. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Which makes me think... maybe let's add this attribute to the login page?

Reply:
I think we can add it to the list of follow up stuff to be done, so that this gets out there and maybe we can get additional feedback from the community. After this is merged I'll create an issue for them.

Reply:
Sounds good.